pyspark/bigdl/keras/converter.py

```diff
@@ -1030,21 +1025,30 @@ def create_atrousconvolution2d(self, klayer, kclayer):
             raise Exception("Please set `bias=True` for AtrousConvolution2D")
         input_shape = klayer.get_input_shape_at(0)
 
-        # TODO: border_mode=`same`
-        if klayer.border_mode == "same":
-            raise Exception("Unsupported border mode: %s" % klayer.border_mode)
-        bpadW, bpadH = self.to_bigdl_2d_padding(klayer.border_mode)
+        h = input_shape[2]
+        w = input_shape[3]
+        kh = config["nb_row"]
+        kw = config["nb_col"]
+        dh = config["subsample"][0]
+        dw = config["subsample"][1]
+        dilation_h = config["atrous_rate"][0]
+        dilation_w = config["atrous_rate"][1]
+        pad_h = 0
+        pad_w = 0
+        if config["border_mode"] == "same":
```

Comment:
extract this as a function?

Reply:
I consider extracting it as a function, but this function will need too many arguments but only does one calculation?